.github/workflows/ci.yml

```
@@ -30,10 +30,10 @@ jobs:
       - name: Send repository dispatch
         if: ${{ github.event_name == 'push' && github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
```

```suggestion
        if: ${{ runner.os == 'Windows' && github.event_name == 'push' && github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
```

We only want to trigger repository dispatch for Windows.